string-manipulation-3/src/reverse-words.js

```diff
@@ -0,0 +1,17 @@
+/* exported reverseWords */
+function reverseWords(string) {
+  var stringArray = string.split(' ');
+  var wordArray = [];
+  var wordString = '';
+  var concatLetters = '';
+  var j;
+  for (var i = 0; i < stringArray.length; i++) {
+    wordString = (stringArray[i]);
```

Comment:
I don't think this `wordString` variable is really necessary, you can just use `stringArray[i]`